src/components/vuestic-components/va-avatar/VaAvatar.vue

```diff
@@ -1,26 +1,84 @@
 <template>
-  <div class="va-avatar">
-    <slot/>
+  <div class="va-avatar" ref="avatar" :style="computedStyle">
+    <va-icon v-if="icon" :name="icon" />
+    <img v-else-if="src" :src="src" />
+    <span v-else><slot/></span>
   </div>
 </template>
 
 <script>
+import { SizeMixin } from '../../../mixins/SizeMixin'
+import { ColorThemeMixin } from '../../../services/ColorThemePlugin'
+import VaIcon from '../va-icon/VaIcon'
+
 export default {
   name: 'va-avatar',
+  mixins: [SizeMixin, ColorThemeMixin],
+  components: {
+    VaIcon,
+  },
+  props: {
+    textColor: {
+      type: String,
+    },
+    square: {
+      type: Boolean,
+      default: false,
+    },
+    icon: {
+      type: String,
+    },
+    src: {
+      type: String,
+    },
+    fontSize: {
+      type: String,
+    },
+  },
+  methods: {
+    udpdateFontSize () {
+      this.$refs.avatar.style.fontSize = `${this.$refs.avatar.offsetHeight * 0.6}px`
+    },
+  },
+  mounted () {
+    if (!this.fontSize) {
+      this.udpdateFontSize()
+    }
+  },
+  computed: {
+    computedStyle () {
+      return {
+        color: this.textColor ? (this.$themes[this.textColor] || this.computeColor(this.textColor)) : '#ffffff',
+        backgroundColor: this.$themes[this.color] || this.colorComputed,
+        borderRadius: this.square ? 0 : '50%',
+        fontSize: this.fontSize || '',
+        ...this.getPropSize('width'),
+        ...this.getPropSize('height'),
+      }
+    },
+
+  },
 }
 </script>
 
 <style lang="scss">
 .va-avatar {
   display: flex;
-  width: 48px;
-  height: 48px;
-  border-radius: 50%;
   overflow: hidden;
+  align-items: 'center';
```

Comment:
Pretty sure it should be center, not 'center'.

Reply:
fixed